src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Averin";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+#myInput {
```

Comment:
[Checklist rule 1](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Fix it everywhere.

Reply:
Use classes only